dmv/test/dmv_test.js

```diff
@@ -13,6 +13,18 @@ describe('DMV', () => {
     })
   })
 
+  it('calling nextCustomer w/o enter throws error', () => {
+    var dmv = new DMV(["Surly Suneel", "Angry Angelica", "Peeved Petra"]);
+    expect(dmv.nextCustomer.bind(dmv)).to.throw('No customers')
+    expect(() => dmv.nextCustomer()).to.throw('No customers')
```

Comment:
Are we having unnecessary repetition here?

Reply:
Agree. Unnecessary repetition here.

Reply:
Please provide another comment to demonstrate "Speak fluently using object-oriented / coding terminology; take code and describe it using words like method, parameter, declare, instantiate...etc"